ga4gh/datarepo.py

```diff
@@ -332,6 +339,11 @@ def verify(self):
         # have verbosity levels. We should provide a way to configure
         # where we look at various chromosomes and so on. This will be
         # an important debug tool for administrators.
+        for ontologyTermMap in self.getOntologyTermMaps():
+            print(
+                "Verifying OntologyTermMap", ontologyTermMap.getLocalId(),
+                "@", ontologyTermMap.getDataUrl())
+            # TODO how do we verify this? Check some well-know SO terms?
```

Comment:
I imagine we would fail somewhere earlier if the file is not formatted correctly. Knowing that for each key in the ontology term map there exists a term would probably be sufficient for now.